model/writer.go

```diff
@@ -628,13 +628,16 @@ func procPage(p *PdfPage) {
 	}
 
 	// Add font as needed.
-	f := DefaultFont()
-	p.Resources.SetFontByName("UF1", f.ToPdfObject())
+	fontName := core.PdfObjectName("UF1")
```

Comment:
Problem description:
> If you open this PDF with the editor and search for UF1 you'll see something like: <</F1 6 0 R/F2 11 0 R/Font2 16 0 R/UF1 0 0 R>>
> It should be /UF1 16 0 R
> Because the writer has the objectsMap dictionary which acts like a sort of cache, and it already has the the Fonts dictionary, it does not process it to resolve references, so it remains as is.

Maybe need to call procPage before page.ToPdfObject in the writer's AddPage function ?

Reply:
I think it's a good idea to call procPage before page.ToPdfObject. However, it does not seem to solve the issue. The resulting PDF still contains /UF1 0 0 R.